utilities/blob_db/blob_file.h

```diff
@@ -26,6 +27,9 @@ class BlobFile {
   friend struct BlobFileComparator;
   friend struct BlobFileComparatorTTL;
 
+ public:
+  using SstFileSet = std::unordered_set<uint64_t>;
```

Comment:
I'm picky again but this alias may not be the usage that is recommended in Google C++ Style: https://google.github.io/styleguide/cppguide.html#Aliases